README.md

```diff
@@ -47,6 +47,8 @@ source tree), the backup is considered *complete*.
 
     conserve backup /backup/home.cons ~
 
+Optionally provide as third parameter a comma separated list of [glob patterns](https://en.wikipedia.org/wiki/Glob_(programming) to exclude files from backup.
```

Comment:
I think I'd rather a `--exclude` option rather than a third parameter, as it can more easily apply to other commands, and is more consistent with eg rsync and duplicity.

Reply:
Also please wrap the text in this file.